bundles/org.openhab.binding.knx/src/main/resources/OH-INF/config/channelConfig.xml

```diff
@@ -27,7 +27,7 @@
 		</parameter>
 		<parameter name="position" type="text" required="false">
 			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to set the absolute position of the dimmer</description>
+			<description>The group address(es) in Group Address Notation to set the absolute brightness</description>
```

Comment:
see above

Reply:
removed from PR, porting https://github.com/smarthomej/addons/pull/257 will override it anyway